### PR TITLE
feat(kousa): add emoji chat token

### PR DESCRIPTION
**Pull request by amitojsingh366**
@goldyydev needs this since he's adding twemoji. he is still confused on weather he should use 2 diff types for custom emotes and twemojis or just use one. @benawad please advise

### CODE REVIEW

File: kousa/lib/broth/message/_types/chat_token.ex [hunk 1]
```diff
@@ -62,4 +63,29 @@ defmodule Broth.Message.Types.ChatToken do
...
...F191}-\x{1F19A}\x{1F201}\x{1F21A}\x{1F22F}\x{1F232}-\x{1F236}\x{1F238}-\x{1F23A}\x{1F250}\x{1F251}\x{1F300}-\x{1F320}\x{1F32D}-\x{1F335}\x{1F337}-\x{1F37C}\x{1F37E}-\x{1F384}\x{1F386}-\x{1F393}\x{1F3A0}-\x{1F3C1}\x{1F3C5}\x{1F3C6}\x{1F3C8}\x{1F3C9}\x{1F3CF}-\x{1F3D3}\x{1F3E0}-\x{1F3F0}\x{1F3F8}-\x{1F407}\x{1F409}-\x{1F414}\x{1F416}-\x{1F43A}\x{1F43C}-\x{1F43E}\x{1F440}\x{1F444}\x{1F445}\x{1F451}-\x{1F465}\x{1F46A}\x{1F479}-\x{1F47B}\x{1F47D}-\x{1F480}\x{1F484}\x{1F488}-\x{1F48E}\x{1F490}\x{1F492}-\x{1F4A9}\x{1F4AB}-\x{1F4FC}\x{1F4FF}-\x{1F53D}\x{1F54B}-\x{1F54E}\x{1F550}-\x{1F567}\x{1F5A4}\x{1F5FB}-\x{1F62D}\x{1F62F}-\x{1F634}\x{1F637}-\x{1F644}\x{1F648}-\x{1F64A}\x{1F680}-\x{1F6A2}\x{1F6A4}-\x{1F6B3}\x{1F6B7}-\x{1F6BF}\x{1F6C1}-\x{1F6C5}\x{1F6D0}-\x{1F6D2}\x{1F6D5}-\x{1F6D7}\x{1F6EB}\x{1F6EC}\x{1F6F4}-\x{1F6FC}\x{1F7E0}-\x{1F7EB}\x{1F90D}\x{1F90E}\x{1F910}-\x{1F917}\x{1F91D}\x{1F920}-\x{1F925}\x{1F927}-\x{1F92F}\x{1F93A}\x{1F93F}-\x{1F945}\x{1F947}-\x{1F976}\x{1F978}\x{1F97A}-\x{1F9B4}\x{1F9B7}\x{1F9BA}\x{1F9BC}-\x{1F9CB}\x{1F9D0}\x{1F9E0}-\x{1F9FF}\x{1FA70}-\x{1FA74}\x{1FA78}-\x{1FA7A}\x{1FA80}-\x{1FA86}\x{1FA90}-\x{1FAA8}\x{1FAB0}-\x{1FAB6}\x{1FAC0}-\x{1FAC2}\x{1FAD0}-\x{1FAD6}]
```
Comment (benawad) [inline kousa/lib/broth/message/_types/chat_token.ex]: I like having this separate emoji type, but I don't think we need to regex it. If someone gives us an invalid emoji we just won't render it on the frontend

**Review by benawad — CHANGES_REQUESTED** [review]

File: kousa/lib/broth/message/_types/chat_token.ex (hunk 1 above)
Comment (amitojsingh366) [reply kousa/lib/broth/message/_types/chat_token.ex]: ohk lol

Comment (goldyydev) [reply kousa/lib/broth/message/_types/chat_token.ex]: once im done with the frontend, ill make sure it renders invalid emojis as text

Comment (amitojsingh366) [reply kousa/lib/broth/message/_types/chat_token.ex]: ok done

**Pull request merged**